tests/test_cnn.py

```diff
@@ -25,14 +25,25 @@ def test_cnn(tmp_path, model_class):
     else:
         # Avoid memory error when using replay buffer
         # Reduce the size of the features
-        kwargs = dict(buffer_size=250, policy_kwargs=dict(features_extractor_kwargs=dict(features_dim=32)))
+        kwargs = dict(
+            buffer_size=250,
+            policy_kwargs=dict(features_extractor_kwargs=dict(features_dim=32)),
+            seed=1,
+        )
     model = model_class("CnnPolicy", env, **kwargs).learn(250)
 
     # FakeImageEnv is channel last by default and should be wrapped
     assert is_vecenv_wrapped(model.get_env(), VecTransposeImage)
 
     obs = env.reset()
 
+    # Test stochastic predict with channel last input
+    if model_class == DQN:
+        model.exploration_rate = 0.9
+
+    for _ in range(10):
+        model.predict(obs, deterministic=False)
```

Comment:
A curiosity: Why 10 steps, and why not 10 steps for deterministic=True? Just to be on save side we could do 10 steps for the True case as well (in case first `predict` modifies something and breaks next one)

Reply:
pretty arbitrary, this is mostly to avoid any regression on the bug I fixed for DQN, and as I set the exploration rate to 0.9, I still to sample the predict method enough times to be sure that the random action part will be called...